src/main/java/com/teamwizardry/librarianlib/features/animator/animations/BasicAnimation.kt

```diff
@@ -33,4 +33,23 @@ class BasicAnimation<T : Any>(target: T, property: AnimatableProperty<T>) : Anim
         val new = lerper.lerp(from, to, progress)
         property.set(target, new)
     }
+
+    // builder methods
+
+    @JvmName("from")
+    fun fromValue(value: Any): BasicAnimation<T> {
+        this.from = value
+        return this
+    }
+
+    @JvmName("to")
+    fun toValue(value: Any): BasicAnimation<T> {
+        this.to = value
+        return this
+    }
+
+    fun ease(easing: Easing): BasicAnimation<T> {
+        this.easing = easing
+        return this
+    }
```

Comment:
```suggestion
    }
    //endregion
```
(with the other suggestion to open the region of course)